rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,9 @@ toc::[]
 [id="rosa-q2-2024_{context}"]
 === Q2 2024
 
+* **Approved Access for ROSA clusters.** Red{nbsp}Hat Site Reliability Engineering (SRE) managing and proactively supporting ROSA Clusters will typically not require access to customer Data as part of the normal operations. In the unlikely event should you need access to customer data, the *Approved Access* functionality provides an interface for customers to review and *approve* or *deny* access requests.
```

Comment:
... In the unlikely event should **Red Hat SRE** need access to customer data ...